mkdocs.yml

```
@@ -52,6 +52,7 @@ nav:
       - Job array: runjobs/scheduled-jobs/throughput.md
       - Interactive jobs: runjobs/scheduled-jobs/interactive.md
       - Container jobs: runjobs/scheduled-jobs/container-jobs.md
+      - FUSE mounts: runjobs/scheduled-jobs/FUSE.md
       - Julia scheduled jobs: runjobs/scheduled-jobs/julia.md
```

Heidi and me discussed to place it under storage, say:
```suggestion
- Storage
  - Storage formats
    - SquashFS: runjobs/scheduled-jobs/FUSE.md
```

---

Text itself is fine.